### PR TITLE
feat: add hasError method

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#7

### CODE REVIEW

**Review by WoodNeck — APPROVED** [review]
LGTM

File: src/ImReadyManager.ts [hunk 1]
```diff
@@ -131,6 +131,13 @@ class ImReadyManager extends Component<ImReadyEvents> {
   public isReady() {
     return this.elementInfos.every(info => info.isReady);
   }
+  /**
+   * Whether an error occurred during check.
+   * @ko 체크하는 동안 에러가 났는지 여부
+   */
+  public hasError() {
```
Comment (mixed) [inline src/ImReadyManager.ts:138]: Do you need a hasError method at other point than preReady or ready?

Comment (daybrush) [reply src/ImReadyManager.ts:138]: @mixed 

It is required when you want to check in somewhere other than the event.

Comment (mixed) [reply src/ImReadyManager.ts:138]: Yes. I think you may need method. Is there a situation that is not preReady or ready as a precondition? Is there any reason to use it without precondition?

**Review by mixed — APPROVED** [review]
LGTM

**Pull request merged**